unique_path_lc62.py

```diff
@@ -0,0 +1,14 @@
+"""
+Name: Srinidhi Bhat
+Did it run on LC: No, TLE 
+
+Time Complexity:O(MXN) - O(2^(max(m,n))) - Each call you have two choices - either go down or right
+                                          - We can keep doing this, till either one of the dimension gets exhausted 
+Space Complexity: O(MXN) - Recursion Stack (Implicit)
+"""

```

Comment:
Find out where the repeated subproblem is and Try to come up with the optimized approach. If you still find it difficult. Refer to the solution taught in class and try it out